spinnaker-monitoring/spectator_client.py

```diff
@@ -33,6 +33,7 @@
 
 
 CONFIG_DIR = '/opt/spinnaker-monitoring/config'
+SOURCES_DIR = '/opt/spinnaker-monitoring/sources'
```

Comment:
Right - we tried to mount a dir within a mounted dir (both /config and /config/sources), and the second mount's files never showed up.